src/main/java/org/team1540/bobafett/commands/elevator/ElevatorPID.java

```diff
@@ -0,0 +1,26 @@
+package org.team1540.bobafett.commands.elevator;
+
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ElevatorPID extends CommandBase {
```

Comment:
I'm not sure this needs to be a command instead of just a method of the subsystem, it's just setting the PID's setpoint